packages/amplify-provider-awscloudformation/src/iterative-deployment/deployment-manager.ts

```diff
@@ -154,19 +174,28 @@ export class DeploymentManager {
       const service = interpret(machine)
         .onTransition(async state => {
           if (state.changed) {
-            this.spinner.render();
             maxDeployed = Math.max(maxDeployed, state.context.currentIndex + 1);
-            if (state.matches('idle')) {
-              this.logSpinnerMessage('deploy', `Starting deployment`);
-            } else if (state.matches('deploy.waitForTablesToBeReady')) {
-              this.logSpinnerMessage('deploy', `Waiting for DynamoDB indices to be ready`);
-            } else if (state.matches('deploy')) {
-              this.logSpinnerMessage('deploy', `Deploying (${maxDeployed} of ${state.context.stacks.length})`);
-            } else if (state.matches('rollback')) {
-              this.logSpinnerMessage('deploy', `Rolling back (${maxDeployed - state.context.currentIndex} of ${maxDeployed})`);
+
+            const key = Object.keys(deploySpinnerMessage).find(elm => state.matches(elm));
```

Comment:
`deploySpinnerMessage` not being plural makes this piece hard to parse on first attempt.